scripts/benchmark/utils/chain_plumbing.py

```diff
@@ -1,3 +1,9 @@
+from pathlib import (
+    Path,
+)
+
```

Comment:
*nitpick:* line breaks should only appear between the three groups: stdlib, 3rd party, own but not within imports of each group.